modeUtil.py

```diff
@@ -5,3 +5,10 @@ def checkOrderById(arrangeMode):
             return True
         else:
             return False
+
+    @staticmethod
+    def checkConditionAssigned(arrangeMode):
+        if arrangeMode in [3,4,5,7]:
```

Comment:
Because the following case mode 7 will also use condition. 
So I added 7 for the future modification.